csm/core/controllers/users.py

```diff
@@ -141,10 +117,13 @@ async def get(self):
     """
     POST REST implementation for creating a csm user
     """
+    @CsmAuth.public
     @CsmAuth.permissions({Resource.USERS: {Action.CREATE}})
     async def post(self):
-        Log.debug(f"Handling users post request."
-                  f" user_id: {self.request.session.credentials.user_id}")
+        Log.debug("Handling users post request.")
+
+        creator = self.request.session.credentials.user_id if self.request.session else None
```

Comment:
Is there any particular handling to be done if `creator` equals `None`?

Reply:
Yes, the service needs to know who attempts to create the user.

Reply:
Oh, OK.  I was asking if we had to handle this special case here due potential errors.  But I noticed that the creator can be `None` in case we are creating the first admin account.  Please confirm if both are correct and I will resolve this thread.

Reply:
Confirm.